jekyll/_cci2/linux-cuda-images-support-policy.adoc

```diff
@@ -0,0 +1,108 @@
+
+[#tagging]
+== Tagging
+
+Tagging for these images will be categorized with the following image slugs: 
+
+* `linux-cuda-<CUDA-version>:<year.month.patch>`
+** For example, `linux-cuda-11:2023.02.1`
+** The “default” slugs will simply remove the tag if users want to keep up-to-date:
+*** For example, `linux-cuda-11` or `linux-cuda-12`
+* `linux-cuda-<CUDA-version>:edge`
+** This will be available a week prior to general releases. These images incorporate any new changes for customers who want the latest version as fast as possible.
+
+
+[#critical-cve-patches]
+== Critical CVE patches
+
+When critical CVEs are disclosed that affect the versions of the operating system or software stack in either Linux or Windows images, we will investigate the impact that this has on our images being used within the CircleCI execution environment. 
+
+In most cases, due to the ephemeral and isolated nature of the environment, it is not necessary to patch these images. We will always communicate our stance on these disclosures via our link:https://discuss.circleci.com/[Discuss Forum].
+
+[#bug-reports-issues-and-prs]
+== Bug reports, issues, and PRs
+
+Please file a link:https://support.circleci.com/hc/en-us/requests/new[support ticket] with CircleCI Support for any issues or bugs found with the Linux CUDA images. Our support team will be able to escalate issues internally to the correct engineering team and provide updates on the issue.
+
+[#image-lifespan-eol]
+== Image lifespan / EOL
+
+Image lifespan will generally follow Nvidia’s CUDA release cycle. 
+
+Once a new major version of CUDA is released, we will begin the deprecation process of the oldest major version that we support and schedule it to be removed. This allows us to maintain two major versions of CUDA effectively.
+
+Example: When CUDA 12.0 is released:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+| Version
+| Support
+
+| CUDA 12
+| `linux-cuda-12` image added
+
+| CUDA 11
+| No change
+
+| CUDA 10
+| `linux-cuda-10` image deprecated
+|===
+
+Example: When CUDA 11.8 is released:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+| Version
+| Support
+
+| CUDA 11.8
+| Added to `linux-cuda-11`
+
+| CUDA 11.6
+| Remains in the image
+
+| CUDA 11.4
+| Remains in the image; becomes “oldest”
+
+| CUDA 11.2
+| Removed from `linux-cuda-11`
+|===
+
+Once a new even numbered minor version is released, that version will be added and the oldest existing version will be removed, provided that more than 3 minor versions exist.
+
+When an image is selected for deprecation and removal, we will create an announcement on our Discuss forum, along with reaching out via email to developers who have requested one of the deprecated images in their recent jobs. We will also plan brownouts to ensure users are aware of the approaching removal of deprecated images.
```

Comment:
```suggestion
When an image is selected for deprecation and removal, we will create an announcement on our Discuss forum, along with reaching out via email to developers who have requested one of the deprecated images in their recent jobs. We will also plan brownouts to help ensure you are aware of the approaching removal of deprecated images.
```

Reply:
Are there in-app banners? @JalexChen

Reply:
@rosieyohannan - yes, the idea is to have in-app banners right before/during brownouts. i'll need to coordinate with execution/fleets on this